### PR TITLE
Update and fix generator for Volto 17 - 8.x.x series.

### PR DESCRIPTION
**Pull request by sneridagh**

### CODE REVIEW

**Review by davisagli — APPROVED** [review]

**Comment by davisagli** [comment]
@sneridagh hmm, there are some failing generator tests

File: packages/generator-volto/README.md
Comment (stevepiercy) [inline packages/generator-volto/README.md:18]: I'd only add known good versions, as who knows what will happen in the future?

```suggestion
| 9.x               | 18.x.x        |
```

File: packages/generator-volto/README.md
Comment (stevepiercy) [inline packages/generator-volto/README.md:24]:
```suggestion
### Create a new Volto project using `npm init`
```

File: packages/generator-volto/README.md
Comment (stevepiercy) [inline packages/generator-volto/README.md:133]:
```suggestion
### Enable an existing add-on as a theme add-on
```

File: packages/generator-volto/README.md
Comment (stevepiercy) [inline packages/generator-volto/README.md:135]:
```suggestion
If you want one of your add-ons to be a theme, you can run this template on top of your add-on.
```

File: packages/generator-volto/README.md
Comment (stevepiercy) [inline packages/generator-volto/README.md:179]: Latest release is 6.0.8.

```suggestion
docker run -it --rm --name=plone -p 8080:8080 -e SITE=Plone -e PROFILES="plone.volto:default-homepage" plone/plone-backend:6.0.8
```

File: packages/generator-volto/README.md
Comment (stevepiercy) [inline packages/generator-volto/README.md:182]:
```suggestion
Consult the [Plone frontend Volto documentation](https://6.docs.plone.org/volto/index.html) for further details.
```

File: packages/generator-volto/README.md
Comment (stevepiercy) [inline packages/generator-volto/README.md:184]: Merged inline above.

```suggestion
```

File: packages/generator-volto/news/5465.bugfix
Comment (stevepiercy) [inline packages/generator-volto/news/5465.bugfix:1]:
```suggestion
Fixed and updated the generator for Volto 17, the 8.x.x series. @sneridagh
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Minor doc fixes.

File: packages/generator-volto/README.md
Comment (stevepiercy) [reply packages/generator-volto/README.md:18]: Actually, this is getting merged to `17.x.x`, so should this line even be here?

**Review by stevepiercy — COMMENTED** [review]
I did not submit this before merge. It's not required, but it would be good to have a reference in the docs instead of updating this section in every branch going forward.

**Pull request merged**
